process/smartContract/process.go

```diff
@@ -821,21 +815,14 @@ func (sc *scProcessor) isCrossShardESDTTransfer(tx data.TransactionHandler) bool
 }
 
 // ProcessIfError creates a smart contract result, consumed the gas and returns the value to the user
-func (sc *scProcessor) ProcessIfError(
-	acntSnd state.UserAccountHandler,
-	txHash []byte,
-	tx data.TransactionHandler,
-	returnCode string,
-	returnMessage []byte,
-	snapShot int,
-) error {
-	err := sc.accounts.RevertToSnapshot(snapShot)
+func (sc *scProcessor) ProcessIfError(acntSnd state.UserAccountHandler, txHash []byte, tx data.TransactionHandler, returnCode string, returnMessage []byte, snapshot int, gasLocked uint64) error {
```

Comment:
Suggest to restore formatting of one parameter per line.

Reply:
done